src/pyhf/pdf.py

```diff
@@ -402,7 +418,21 @@ def expected_data(self, pars):
 
 
 class Model(object):
+    """The main pyhf model class."""
+
     def __init__(self, spec, batch_size=None, **config_kwargs):
+        """
+        Construct a pyhf Model.
+
+        Args:
+            spec (`jsonable`): a json specification
```

Comment:
Probably have consistent capitalization on the docstring explanation. (Where I vote for using upper case not lower case as the standard)